extensions/typescript-language-features/src/features/fileConfigurationManager.ts

```diff
@@ -185,6 +185,7 @@ export default class FileConfigurationManager extends Disposable {
 		return {
 			quotePreference: this.getQuoteStylePreference(config),
 			importModuleSpecifierPreference: getImportModuleSpecifierPreference(config),
+			importModuleSpecifierEnding: getImportModuleSpecifierEndingPreference(config),
```

Comment:
Since this change relies on a protocol changes from TS 3.9 but VS Code will be building against TS 3.8 for a few more iterations, you'll need to add some types casts to make `Proto.UserPreferences` happy with the additional property